HIRS_AttestationCAPortal/src/main/java/hirs/attestationca/portal/util/CertificateStringMapBuilder.java

```diff
@@ -56,9 +63,30 @@ private CertificateStringMapBuilder() {
                 data.put("isSelfSigned", "false");
             }
 
+            data.put("authKeyId", certificate.getAuthKeyId());
+            if (certificate.getAuthInfoAccess() != null) {
+                data.put("authInfoAccess", certificate.getAuthInfoAccess());
+            }
+            data.put("crlPoints", certificate.getCrlPoints());
```

Comment:
Does there need to be a `null` check for this field? Doesn't look like it exists for `AttributeCertificates`. That being said, it looks like you just avoid rendering it in the JSP if it is null.

Reply:
Moved this line to the EK and CA set up sections.